### PR TITLE
Added DB connection parameters

### PR DESCRIPTION
**Pull request by ilinic**
added connection parameters required by MySQL 8 and upgraded debezium version

### CODE REVIEW

File: misc/tb/src/TB.java [hunk 1]
```diff
@@ -86,6 +86,11 @@ public static void main(String[] args) throws Exception {
         config.setProperty("database.password", ns.getString("password"));
         config.setProperty("database.server.name", "tb");
         config.setProperty("database.history", "io.debezium.relational.history.FileDatabaseHistory");
+
+        config.setProperty("database.allowPublicKeyRetrieval", "true");
+        config.setProperty("log_bin", "mysql-bin");
+        config.setProperty("binlog_format", "row");
```
Comment (benesch) [inline misc/tb/src/TB.java:92]: I'm a bit confused. Aren't `log_bin` and `binlog_format` MySQL system configuration parameters that need to be set in `my.cnf`, not in Debezium connection parameters?

File: misc/tb/src/TB.java [hunk 2]
```diff
@@ -86,6 +86,11 @@ public static void main(String[] args) throws Exception {
         config.setProperty("database.password", ns.getString("password"));
         config.setProperty("database.server.name", "tb");
         config.setProperty("database.history", "io.debezium.relational.history.FileDatabaseHistory");
+
+        config.setProperty("database.allowPublicKeyRetrieval", "true");
```
Comment (benesch) [inline misc/tb/src/TB.java:90]: Should this go in the `type.equals("mysql")` block below? It's only valid for MySQL, right?

**Review by benesch — COMMENTED** [review]
Sorry for the delay. I just did a quick review and left a few notes within.

File: misc/tb/src/TB.java (hunk 1 above)
Comment (ilinic) [reply misc/tb/src/TB.java]: Sorry for that - initially I just wanted to make it work locally and did not think about publishing changes

File: misc/tb/src/TB.java (hunk 2 above)
Comment (ilinic) [reply misc/tb/src/TB.java]: implemented - thanks

**Comment by benesch** [comment]
Awesome, thank you! I cleaned up some of the whitespace and will merge in #8222.

**Pull request closed without merging**